tests/debug/compare_lib.py

```diff
@@ -52,7 +52,9 @@ def compare(
         super().compare(name, ref_args, new_args, ret_indices)
 
         if self.time_eval and name not in self.time_eval_results:
-            res = self.mod.time_evaluator(name, self.device)(*new_args)
+            res = self.mod.time_evaluator(name, self.device, number=100, repeat=3)(
+                *new_args
+            )
```

Comment:
Per experiments, `number=100, repeat=3` gives evaluation results closer to real e2e time.